apps/opik-backend/src/main/java/com/comet/opik/infrastructure/http/CorsFactory.java

```diff
@@ -13,6 +13,10 @@
 
 @Slf4j
 public class CorsFactory {
+    public static final String COMET_SDK_API_HEADER = "Comet-Sdk-Api";
+    public static final String COMET_USERNAME_HEADER = "comet-username";
+    public static final String COMET_REACT_VER_HEADER = "comet-react-ver";
+    public static final String COMET_WORKSPACE_REQUEST_HEADER = "comet-workspace";
```

Comment:
I'm not it makes a difference here, but in `RequestContext.WORKSPACE_HEADER`, the header is `Comet-Workspace`

Reply:
Apart from aligning the value, +1 Let's reuse the existing constants as much as possible. Also, reduce their visibility as much as you can, private if possible.

Reply:
Done in https://github.com/comet-ml/opik/pull/520/commits/ef613aa9fa1370183c9a334539bbbe2a6b4bbe01